md4c/md4c.c

```diff
@@ -2696,7 +2696,8 @@ md_build_mark_char_map(MD_CTX* ctx)
     ctx->mark_char_map['!'] = 1;
     ctx->mark_char_map[']'] = 1;
     ctx->mark_char_map['\0'] = 1;
-
+	if (ctx->r.flags & MD_FLAG_REDDITSLASHDETECTION)
```

Comment:
Uff, indentation using tab chars here and on some other places of your patch. Please use just spaces as in the rest of the source file(s).